src/js/components/homepage/hero/TotalAmount.jsx

```diff
@@ -25,13 +28,16 @@ const TotalAmount = ({
 }) => {
     const ref = useRef(null);
 
-    useEffect(() => (
-        () => {
+    useEffect(() => {
+        if (skipAnimation) {
+            completeIncrement();
+        }
+        return () => {
             if (amountUpdate) {
                 window.clearTimeout(amountUpdate);
             }
-        }
-    ), []);
+        };
+    }, []);
 
     useEffect(() => {
         const updateAmount = (amount, speedOfUpdate) => new Promise((resolve) => {
```

Comment:
We can move this updateAmount into the conditional below `(!isLoading && !skipAnimation)` since if we are skipping the animation this never needs to be read.